pom.xml

```diff
@@ -753,6 +762,35 @@ under the License.
 				<extensions>true</extensions>
 			</plugin>
 
+			<plugin>
+				<groupId>org.codehaus.mojo</groupId>
+				<artifactId>exec-maven-plugin</artifactId>
+				<version>3.1.0</version>
+				<inherited>false</inherited>
```

Comment:
We dont want to run this multiple times. The checker is not written to only consider part of the project.